test/sdn_migration/sdn_migration_test.go

```diff
@@ -4,27 +4,27 @@ import (
 	"context"
 	"encoding/json"
 	"fmt"
+	"github.com/go-logr/logr"
```

Comment:
third party imports shouldn't be with stdlib imports, this should get fixed if you format the file